.github/workflows/system-tests.yml

```
@@ -111,6 +109,8 @@ jobs:
     name: Build (${{ matrix.app }})
     permissions:
       packages: write
+    outputs:
+      FORCED_TESTS_LIST_JSON: ${{steps.read_forced_tests_list.outputs.FORCED_TESTS_LIST_JSON}}
```

As a good practice - don't include the data structure in the name, because if structure change it will force the name change.

```suggestion
      FORCED_SYSTEM_TESTS: ${{steps.read_forced_tests_list.outputs.FORCED_TESTS_LIST_JSON}}
```

---

Fixed in ad349271a55222d69c2b9b5d40d8ef36f52d5ee9